src/extension-runners/chromium.js

```diff
@@ -150,8 +150,28 @@ export class ChromiumExtensionRunner {
       });
     });
 
+    const chromeFlags = [...DEFAULT_CHROME_FLAGS];
+    let startingUrl;
+    var specialStartingUrls = '';
+    if (this.params.startUrl) {
+      const startingUrls = Array.isArray(this.params.startUrl) ?
+        this.params.startUrl : [this.params.startUrl];
+
+      // Remove URLs starting with chrome:// from startingUrls and let bg.js open them instead
+      for (let i = 0; i < startingUrls.length; i++) {
+        if (startingUrls[i].toLowerCase().startsWith('chrome://')) {
+          specialStartingUrls += startingUrls[i] + " "
+          startingUrls.splice(i, 1)
```

Comment:
By removing the element during the iteration, you'll end up skipping URLs.

For example:

```
[
  "chrome://extensions",
  "chrome://version"
]
```

At `i=0`, `"chrome://extensions"` would be appended to the list of start URLs.
But due to the `startingUrls.splice(i, 1)` call, the array will be `["chrome://version"]`.
At the next iteration, `i=1`, which would result in `"chrome://version"` to be skipped.

To fix this, you could iterate over the URLs in reverse order and prepend the URLs to the `specialStartingUrls` list. Then `.splice(i, 1)` wouldn't be an issue any more.